project/build.properties

```diff
@@ -1 +1 @@
-sbt.version = 1.0.0
+sbt.version = 0.13.16
```

Comment:
No worries, feel free to close, just opened this as an example for a dialogue in the sbt gitter channel.